src/helpers/generate-path-matrix.ts

```diff
@@ -11,34 +11,43 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import * as core from '@actions/core';
 import { chunk, uniq } from 'lodash';
 import { context } from '@actions/github';
 import { getChangedFilepaths } from '../utils/get-changed-filepaths';
 import micromatch from 'micromatch';
 
 export class GeneratePathMatrix {
-  paths = '';
+  paths?: string;
+  globs?: string;
   override_filter_paths?: string;
   override_filter_globs?: string;
   paths_no_filter?: string;
   batches?: string;
 }
 
 export const generatePathMatrix = async ({
-  paths,
+  paths = '',
```

Comment:
Inputs are already defaulted to empty string

Reply:
i changed the interface definition to set `paths`/`globs` as optional since they have an XOR relationship, so i moved their initialization to null string here

Reply:
I'm just saying that `paths = ''` is actually redundant here

Reply:
The `getInput` method in `@actions/core` returns empty string if no value is passed to the action, and that's what we [use](https://github.com/ExpediaGroup/github-helpers/blob/b7e1485b7ba1a900c5dbdc00aa47bffb2649d050/src/utils/get-action-inputs.ts#L25) to read in all inputs dynamically

Reply:
But anyway, I still don't think we should default `paths` or `globs` to empty string to stay consistent with the rest. We should just handle the undefined case properly in the helper I think